### DISCUSSION TITLE
Applying color to footer of meeting details page

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1183] [Q&A]
Can anybody tell me how to apply color to a text I have in the footer of the meeting pages?
I have no experience with CSS, so if anyone can help, please do so in the simplest of ways so I can understand.

Much appreciated!
Nicky

**Comment by anchovie91471** [comment]
Can you proved a link to the site and the specific part of the footer you want to change?

Thanks!

**Comment by OSAwebsite** [comment]
Sure. Have a look at any meeting's details page.
The note I want to show in color is in the footer, where I explain that the old 'request a change to this listing' has been replaced by the new button 'Update meeting info'.

al-anon.alateen.on.ca

I have felt the need to format text in the footer (and the header, for that matter) before, but it was not really super essential.
But with the switch to the new User Interface my fear is that members will get confused by the new button and process.

Thanks!
Nicky

**Comment by anchovie91471** [comment]
To change the text to red, add this to your stylesheet:

```
.meetings-widgets-bottom .widget p {
    color: red;
}
```

**Comment by OSAwebsite** [comment]
I don't mean to sound ignorant but...
Where is this stylesheet?
And will it be overwritten when there is an upgrade in TSML or my theme or
PHP or Wordpress?
I have never touched any of the underlying code, as I know too little about
it...

If I find this sheet, do I add it just anywhere?




-
Yours in service,
Nicky D
Acting Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by anchovie91471** [comment]
No changes to styles will be overwritten by TSML updates. Where to put your styles depends on what version of Wordpress you're using and your theme. 

This is outside the scope of our support since it's not specific to TSML. 

Go to Appearance -> Customize, and there should be (depending on the theme you're using) an "Additional CSS" tab where you can add the rule.

**Comment by OSAwebsite** [comment]
OK, let me give that a try..... thanks for the information!


-
Yours in service,
Nicky D
Acting Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by OSAwebsite** [comment]
Cool, that worked. But now another issue arises:
the remark is appearing in the bottom of the list and of the details page.
I remember reading about that change and I regret it.
Lists have a different purpose in user interfaces than details pages...

I may have to rethink the upgrade to the new UI. Too many drawbacks for my
particular situation....

Thanks for telling me how to do colour though.
Is there also a way to just apply colour to part of the text in that widget
area?



-
Yours in service,
Nicky D
Acting Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*
